README.md

```diff
@@ -1,21 +1,22 @@
-## 18F Engineering Practices
+# Technology Transformation Services (TTS) Engineering Practices
 
-18F promotes team best practices across specialty areas through guilds. These guilds support their members in whatever way deemed most appropriate by those members themselves.
+[TTS](https://www.gsa.gov/about-us/organization/federal-acquisition-service/technology-transformation-services) offices and programs, including [18F](https://18f.gsa.gov), [CoE](https://coe.gsa.gov), [PIF](https://presidentialinnovationfellows.gov), and [Solutions](https://www.gsa.gov/about-us/organization/federal-acquisition-service/technology-transformation-services/tts-solutions), promote team best practices across specialty areas through guilds. These guilds support their members in whatever way deemed most appropriate by those members themselves.
```

Comment:
Should we expand the acronyms for CoE and PIF?

Reply:
I was playing around a bit with the kramdown (Markdown) support for acronym expansion. It doesn't look so good in Chrome, especially when you have an acronym and a link, so I took it out:
<img width="153" alt="Screen Shot 2020-02-03 at 10 41 33 AM" src="https://user-images.githubusercontent.com/37100189/73680894-03ab5e80-4672-11ea-92d9-bd6c562fbd9b.png">

Since I don't refer to them later in the doc and I have links to each of them, I thought I'd try for conciseness. But your point is valid – I should have just stuck to the "expand acronym on first use" rule. I'll make a quick fix – thanks!